CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-# Version set according the the cmake versions available in Ubuntu/Bionic:
-# https://packages.ubuntu.com/bionic/cmake
-cmake_minimum_required(VERSION 3.10.2)
+# Version set according the the cmake versions available in Ubuntu/Focal:
+# https://packages.ubuntu.com/focal/cmake
+cmake_minimum_required(VERSION 3.16.3)
```

Comment:
Is this enough for say Emscripten's requirements (both versions and current builders)?

Reply:
I believe so. I took a look at Emscripten's CI config before making this change and as far as I could tell, it was using focal instead of bionic.